genet/predict/PredUtils.py

```diff
@@ -27,23 +27,55 @@ def preprocess_masked_seq(data, seq_length):
 
 def one_hot_encode(seq):
     mapping = {"A": 0, "C": 1, "G": 2, "T": 3, "X": 4}
-    map_seq = [mapping[i] for i in seq]
+    map_seq = np.array([mapping[i] for i in seq])
```

Comment:
What purpose of this changes? And did you tested if this changes not affect other .predict module such as 'DeepPrime'? Since this one_hot_encode function is used for preprocessing data for DeepPrime module, please check possible bugs in that pipelines. 

Also, using np.array is faster or better than original list comprehension?